project/style.css

```diff
@@ -0,0 +1,227 @@
+.login-section{
+  margin-left: 15%;
+  margin-right:15%;
+  margin-top: 35%;
+}
+
+.section-image1{
```

Comment:
Are we sure we'll only ever have at most 2 of these images? I'm normally wary of including indices in class names, as there are usually nicer ways of pulling off the same thing, though I'm not saying that in certain contexts, this wouldn't be a viable solution.